src/data/finsupp/basic.lean

```diff
@@ -199,6 +199,16 @@ begin
   { rw [single_eq_of_ne h, zero_apply] }
 end
 
+lemma single_of_single_apply (a a' : α) (b : M) :
+  single a ((single a' b) a) = single a' (single a' b) a :=
+begin
+  rw [single_apply, single_apply],
+  ext,
+  split_ifs,
+  { rw h, },
```

Comment:
are those commas OK in mathlib?

Reply:
We're not consistent about them, but I think it's fine.